.github/workflows/ci.yml

```diff
@@ -3,12 +3,22 @@ name: run-tests
 on: [push, pull_request]
 
 jobs:
-  test:
+  tests:
     runs-on: ubuntu-latest
     strategy:
-      fail-fast: true
       matrix:
-        laravel: [6, 8, 9]
+        php: ["7.3", "8.0", "8.1"]
+        laravel: ["^6.0", "^8.0", "^9.0", "^10.0"]
+        include:
+          - laravel: "^9.0"
+            testbench: "^7.0"
+          - laravel: "^10.0"
+            testbench: "^8.0"
+        exclude:
+          - laravel: "^10.0"
+            php: "8.0"
+          - laravel: "^9.0"
+            php: "7.3"
```

Comment:
What's the reason for moving to this syntax?

Reply:
Just to be the same syntax as tenancy :) i can rollback the changes if you wish

Reply:
And what's the reason for the change there?

Reply:
I've added the exclude and include as we don't want to test Laravel 10.x using php 8.0 for example, and same for the others..